contracts/Identity.sol

```diff
@@ -64,24 +64,44 @@ contract Identity is KeyManager {
     address to,
     uint256 value,
     bytes memory data
-  )
+  ) onlyAction
   public
   returns (bool success)
   {
-
-    bytes32 key_ = addressToKey(msg.sender);
-    require(
-      keyHasPurpose(key_, ACTION) && _keys[key_].revokedAt == 0,
-      "Requester must have an ACTION purpose"
-    );
-
     // solium-disable-next-line security/no-inline-assembly
     assembly {
-      // check if contract to be call exists
+      // check if contract to be called exists
       if iszero(extcodesize(to)) {
         revert(0, 0)
       }
       success := call(gas, to, value, add(data, 0x20), mload(data), 0, 0)
```

Comment:
What happens if the contract I want to send some ETH is not `payable`?

Would the complete transaction fail or would the identity contract own the money?

In case the identity contract would own it.
Is it possible to withdraw the ETH from the identity contract?

Reply:
You should know what contract you are calling. The identity contract is just a proxy and does not care about how the contract is calling is implemented. Funds on the identity contract is not something we support and I think it a broader discussion.

Reply:
If the whole transaction fails I am fine with it.  :)